README.md

```diff
@@ -320,6 +320,40 @@ NAME                HOST/PORT                                                  P
 kogito-data-index   kogito-data-index-kogito.apps.mycluster.example.com               kogito-data-index   8080   None
 ```
 
+### Kogito Data Index Integration with persistent Kogito Services
+
+If your Kogito Service has [persistence](https://github.com/kiegroup/kogito-runtimes/wiki/Persistence) enabled, 
```

Comment:
persistence and messaging?

Reply:
Are you sure we also need messaging? Because IIRC the codegen will generate the proto files if the persistence addon is present, no?

Reply:
to generate the proto files we only need persistence, but to integrate at runtime, we need messaging... so depending what is the context you want to describe here.

Reply:
The context is the proto files generation. If there are files, we index them.

Reply:
yep, then its fine as is :)